src/test/scala/zio/sqs/ZioSqsMockServer.scala

```diff
@@ -7,18 +7,20 @@ import zio.aws.sqs.Sqs
 import org.elasticmq.rest.sqs.{ SQSRestServer, SQSRestServerBuilder }
 import software.amazon.awssdk.auth.credentials.{ AwsBasicCredentials, StaticCredentialsProvider }
 import software.amazon.awssdk.regions.Region
-import zio.{ Task, UIO, ZLayer, ZManaged }
+import zio.{ UIO, ZIO, ZLayer }
 
 object ZioSqsMockServer {
   private val staticCredentialsProvider: StaticCredentialsProvider =
     StaticCredentialsProvider.create(AwsBasicCredentials.create("key", "key"))
   private val uri                                                  = new URI("http://localhost:9324")
   private val region: Region                                       = Region.AP_NORTHEAST_2
 
-  val serverResource: ZManaged[Any, Throwable, SQSRestServer] =
-    ZManaged.acquireReleaseWith(
-      Task(SQSRestServerBuilder.start())
-    )(server => UIO.succeed(server.stopAndWait()))
+  val serverResource: ZIO[Any, Throwable, SQSRestServer] =
+    ZIO.scoped {
```

Comment:
I think this one shouldn't be there too, or it will stop right away.

Reply:
Ah ha, that was the source of the failing tests.  Thanks!